client/blocks/eligibility-warnings/index.jsx

```diff
@@ -27,19 +27,34 @@ function getHoldMessages( translate ) {
 			description: '',
 			supportUrl: '',
 		},
+		TRANSFER_ALREADY_EXISTS: {
+			title: translate( 'Installation in progress' ),
+			description: translate( 'Another installation is already in progress.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
 			description: translate( 'This feature is only allowed on sites with a business plan.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
+		NO_JETPACK_SITES: {
+			title: translate( 'Jetpack site' ),
+			description: translate( 'This feature is not supported on Jetpack sites.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'This feature is not supported on VIP sites.' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/eligibility-holds-messages)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).